core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1444,7 +1180,8 @@ oppia.factory('autosaveInfoModalsService', [
             };
 
             $scope.lostChangesHtml = (
-              lostChangesService.makeHumanReadable(lostChanges).html());
+            ChangesInHumanReadableFormService.makeHumanReadable(
+                  lostChanges).html());
```

Comment:
Again, deindent line 1184 by 2 spaces (and reindent line 1183 by 2 spaces as before)